lizard_auth_server/tests/test_models.py

```diff
@@ -254,3 +254,32 @@ def test_profile_properties(self):
         p.first_name
         p.last_name
         p.email
+
+
+class TestProfile(TestCase):
+
+    def test_no_site_access(self):
+        """Test that in the current (base) factory configuration the user
+        profile has no access to the site.
+        """
+        company = factories.CompanyF()
+        profile = factories.ProfileF()
+        site = factories.SiteF.create(available_to=[company])
+        self.assertEqual(profile.has_access(site), False)
+
+    def test_site_access_as_employee(self):
+        """Test that user has access when its company is in the available_to
+        list of the site, i.e., employees have access to their company's sites.
```

Comment:
Dit gedrag moet later *misschien* veranderen in de zin dat het misschien een expliciete keuze wordt om iedereen toegang te geven. En dat je er ook voor kan kiezen individuele gebruikers toegang te geven.

Voorlopig OK.